payments.md

```diff
@@ -24,21 +24,30 @@ payment as a resource represents following information
 	POST /payment
 	Authorization: Bearer {accesstoken}
 	cid: 05e230bf-a9cf-4b31-bc54-d3a995f62526
```

Comment:
Why do we need a CID? Does that have any significance or does it provide any security?